test/gtest/uct/test_atomic_key_reg_rdma_mem_type.cc

```diff
@@ -30,14 +30,41 @@ UCS_TEST_SKIP_COND_P(uct_atomic_key_reg_rdma_mem_type, fadd64,
                      !check_atomics(UCS_BIT(UCT_ATOMIC_OP_ADD), FOP64) ||
                      !check_rdma_memory())
 {
-    mapped_buffer recvbuf(sizeof(uint64_t), receiver(), 0UL,
-                          UCS_MEMORY_TYPE_RDMA);
+    mapped_buffer *recvbuf     = nullptr;
```

Comment:
consider using `std::unique_ptr` instead of raw pointer to automate memory management, and make the code exception-safe

Reply:
done